express/scripts/scripts.js

```diff
@@ -2021,6 +2078,7 @@ async function loadLazy() {
 
   loadBlocks(main);
   loadCSS('/express/styles/lazy-styles.css');
+  loadAdobeCleanHanTypekit();
```

Comment:
Try adding this to `loadDelayed` instead

Reply:
@rofe, where do you refer to by `loadDelayed`? I can't find it anywhere in the code.

Reply:
You're right, this project is using an older version of the page loader. The equivalent would be `loadMartech` here.

Reply:
Done.